### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by wlhlm**
The German translation got spotty, so fix that!

The changes are quite big, so it'd be great if a native speaker could review it.

The second commit adds a missing tooltip.

### CODE REVIEW

File: resources/assets/enderio/lang/de_DE.lang [hunk 1]
```diff
@@ -53,45 +59,117 @@ enderio.itemItemConduit.name=Item-Leitung
...
+item.itemPowerItemFilter.tooltip.detailed.line1=Filtert aufladbare Items basierent darauf
+item.itemPowerItemFilter.tooltip.detailed.line2=wie viel RF diese gespeichert haben
 
 item.itemExtractSpeedUpgrade.name=Item-Leitungsgeschwindigkeits-Upgrade
+item.itemExtractSpeedUpgrade.tooltip.basic.line1=Upgrade für Item-Leitungen
 item.itemExtractSpeedUpgrade.tooltip.detailed.line1=Erhöht Geschwindigkeit um 4
 item.itemExtractSpeedUpgrade.tooltip.detailed.line2=Items pro Extraktion
 item.itemExtractSpeedUpgrade.tooltip.detailed.line3=Maximum der Upgrades ist 15
 
+item.itemExtractSpeedDowngrade.name=ItemßLeitungsgeschwindigkeits-Downgrade
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Item-Leitungsgeschwindigkeitsupgrade

Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Sorry, downgrade.

File: resources/assets/enderio/lang/de_DE.lang [hunk 2]
```diff
@@ -53,45 +59,117 @@ enderio.itemItemConduit.name=Item-Leitung
...
+item.itemExtractSpeedDowngrade.tooltip.basic.line1=Downgrade für Item-Leitungen
+item.itemExtractSpeedDowngrade.tooltip.detailed.line1=Reduziert Geschwindigkeit auf 1
+item.itemExtractSpeedDowngrade.tooltip.detailed.line2=Item pro Extraktion
+ 
+item.itemInventoryPanelUpgrade.name=Fernsteuerungs-Upgrade
+item.itemInventoryPanelUpgrade.tooltip.basic.line1=Inventar-Panelbedienbarkeit
+item.itemInventoryPanelUpgrade.tooltip.detailed.line1=Funktions-Upgrade für Item-Leitungen
+item.itemInventoryPanelUpgrade.tooltip.detailed.line2=Erlaubt dem Netzwerk-angehörigen Inventar-Panels
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Erlaubt den ...

File: resources/assets/enderio/lang/de_DE.lang [hunk 3]
```diff
@@ -53,45 +59,117 @@ enderio.itemItemConduit.name=Item-Leitung
...
 enderio.phasedIronNugget.name=Pulsierender Eisennugget
-tile.blockDarkIronBars.name=Dunkeleisengitter
+tile.blockDarkIronBars.name=Dunkelstahlgitter
+tile.blockEndermanSkull.name=Enderman-Schädel
+tile.blockEndermanSkull.base.name=Enderman-Kopf
+tile.blockEndermanSkull.reanimated.name=Reanimierter Enderman-Kopf
+tile.blockEndermanSkull.tormented.name=Gequälter Enderman-Kopf
 enderio.blastResistant=Explosionssicher
 enderio.lightEmitter=Verbreitet Licht
+enderio.lightBlocker=Blockiert Locht
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Licht

File: resources/assets/enderio/lang/de_DE.lang [hunk 4]
```diff
@@ -53,45 +59,117 @@ enderio.itemItemConduit.name=Item-Leitung
...
+enderio.tooltip.isBeaconBase=Kann als Beacon-Basis genutzt werden
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Leuchtfeuerbasis

File: resources/assets/enderio/lang/de_DE.lang [hunk 5]
```diff
@@ -99,12 +177,17 @@ enderio.machineChassi.name=Maschinenhülle
 enderio.basicGear.name=Basiszahnrad
 enderio.pulsatingCrystal.name=Pulsierender Kristall
 enderio.vibrantCrystal.name=Dynamischer Kristall
+enderio.enderCrystal.name=Enderkristall
+enderio.attractorCrystal.name=Verführender Kristall
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Anziehender Kristall

File: resources/assets/enderio/lang/de_DE.lang [hunk 6]
```diff
@@ -172,6 +273,8 @@ tile.blockSolarPanel.tooltip.detailed.line1=Produziert Energie am Tag
 tile.blockSolarPanel.tooltip.detailed.line2=Braucht klare Sicht
 tile.blockSolarPanel.tooltip.detailed.line3=zum Himmel
 enderio.maxSolorProduction=Max.-Output
+enderio.tooltip.efficiency=Effizienz
+enderio.tooltip.sunlightBlocked=Himmelverdeckt!
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Himmel verdeckt!

File: resources/assets/enderio/lang/de_DE.lang [hunk 7]
```diff
@@ -190,12 +293,44 @@ item.itemLightInverted.name=Licht (Invertiert)
...
+item.itemWirelessLight.tooltip.detailed.line3=Kann Energie von Drahtlos-Ladegeräten empfangen.
+ 
+item.itemWirelessLightInverted.name=Drahtlosgespeistes Licht (Invertiert)
+item.itemWirelessLightInverted.tooltip.detailed.line1=Erleuchtet ein Gebiet, whenn mit
+item.itemWirelessLightInverted.tooltip.detailed.line2=Energie versorgt
+item.itemWirelessLightInverted.tooltip.detailed.line3=Schalte mit einem Redstonesignal ab.
+item.itemWirelessLightInverted.tooltip.detailed.line4=Kann Energie von Drahtlos-Ladegeräten empfangen.
+
+tile.blockCapacitorBank.name=Kondensatorbank (Veraltet)
+tile.blockCapacitorBank.creative.name=Kreativkondensatorbank (Veraltet)
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Kreative Kondensatorbank (Veraltet)

File: resources/assets/enderio/lang/de_DE.lang [hunk 8]
```diff
@@ -190,12 +293,44 @@ item.itemLightInverted.name=Licht (Invertiert)
...
+tile.blockCapacitorBank.tooltip.detailed.line1=In einen neuen Block umwandeln
+tile.blockCapacitorBank.tooltip.detailed.line2=durch Platzieren in Fertigungsgitter.
 tile.blockCapacitorBank.tooltipPrefix=Enthält
 
+tile.blockCapBank.creative.name=Kreativkondensatorbank
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Kreative Kondensatorbank

File: resources/assets/enderio/lang/de_DE.lang [hunk 9]
```diff
@@ -208,10 +343,32 @@ tile.blockSagMill.tooltip.detailed.line4=Ein Upgrade mit Kondensatoren wird
 tile.blockSagMill.tooltip.detailed.line5=Geschw. und Energiespeicher erhöhen
 tile.blockSagMill.tooltipPrefix=Enthält
 
-tile.blockHyperCube.name=Dimensional-Transceiver
+tile.blockHyperCube.name=Dimensionaler Sende-Empfänger (Veraltet)
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Transceiver ist "eingedeutscht" 😏

File: resources/assets/enderio/lang/de_DE.lang [hunk 10]
```diff
@@ -208,10 +343,32 @@ tile.blockSagMill.tooltip.detailed.line4=Ein Upgrade mit Kondensatoren wird
...
 tile.blockHyperCube.tooltip.detailed.line2=drahtlos durch jegliche Dimension und
-tile.blockHyperCube.tooltip.detailed.line3=zwischen Dimensionen
+
+tile.blockTransceiver.name=Dimensionaler Sende-Empfänger
+tile.blockTransceiver.tooltip.detailed.line1=Transportiert Items, Flüssigkeiten und Energie
+tile.blockTransceiver.tooltip.detailed.line2=drahtlos über beliebige Entfernungen
+tile.blockTransceiver.tooltip.detailed.line3=und Dimensionen.
+ 
+tile.blockEnderRail.name=Endergleis
+tile.blockEnderRail.tooltip.detailed.line1=Teleportiert Minecarts wenn platziert
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Teleportiert Loren

File: resources/assets/enderio/lang/de_DE.lang [hunk 11]
```diff
@@ -208,10 +343,32 @@ tile.blockSagMill.tooltip.detailed.line4=Ein Upgrade mit Kondensatoren wird
...
+tile.blockEnderRail.tooltip.detailed.line2=auf einem Dimensionalen Sende-Empfänger
+tile.blockEnderRail.tooltip.detailed.line3=Setze das Ziel im
+tile.blockEnderRail.tooltip.detailed.line4=Gleis-Tab des Sende-Empfänges
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Transceiver

File: resources/assets/enderio/lang/de_DE.lang [hunk 12]
```diff
@@ -208,10 +343,32 @@ tile.blockSagMill.tooltip.detailed.line4=Ein Upgrade mit Kondensatoren wird
 tile.blockSagMill.tooltip.detailed.line5=Geschw. und Energiespeicher erhöhen
 tile.blockSagMill.tooltipPrefix=Enthält
 
-tile.blockHyperCube.name=Dimensional-Transceiver
+tile.blockHyperCube.name=Dimensionaler Sende-Empfänger (Veraltet)
 tile.blockHyperCube.tooltip.detailed.line1=Transportiert Items, Flüssigkeiten und Energie
 tile.blockHyperCube.tooltip.detailed.line2=drahtlos durch jegliche Dimension und
-tile.blockHyperCube.tooltip.detailed.line3=zwischen Dimensionen
+
+tile.blockTransceiver.name=Dimensionaler Sende-Empfänger
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Transceiver

File: resources/assets/enderio/lang/de_DE.lang [hunk 13]
```diff
@@ -208,10 +343,32 @@ tile.blockSagMill.tooltip.detailed.line4=Ein Upgrade mit Kondensatoren wird
...
+tile.blockTransceiver.tooltip.detailed.line2=drahtlos über beliebige Entfernungen
+tile.blockTransceiver.tooltip.detailed.line3=und Dimensionen.
+ 
+tile.blockEnderRail.name=Endergleis
+tile.blockEnderRail.tooltip.detailed.line1=Teleportiert Minecarts wenn platziert
+tile.blockEnderRail.tooltip.detailed.line2=auf einem Dimensionalen Sende-Empfänger
+tile.blockEnderRail.tooltip.detailed.line3=Setze das Ziel im
+tile.blockEnderRail.tooltip.detailed.line4=Gleis-Tab des Sende-Empfänges
+tile.blockEnderRail.tooltip.detailed.line5=Setze Richtung mit Shift-Rechtsklick
+tile.blockEnderRail.tooltip.detailed.line6=durch einen Schraubenschlüssel
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: durch Shift-Rechtsklick mit einem Schraubenschlüssel.

File: resources/assets/enderio/lang/de_DE.lang [hunk 14]
```diff
@@ -238,6 +395,22 @@ tile.blockZombieGenerator.name=Zombie-Generator
...
+tile.blockKillerJoe.name=Killer Joe
+tile.blockKillerJoe.tooltip.detailed.line1=Attackiert nahe Mobs und Spieler,
+tile.blockKillerJoe.tooltip.detailed.line2=wenn mit einem Schwert und
+tile.blockKillerJoe.tooltip.detailed.line3=Nahrhaftem Destillat versorgt.
+tile.blockKillerJoe.tooltip.detailed.line4=Erfahrung kann als Flüssige XP
+tile.blockKillerJoe.tooltip.detailed.line5=abgefüllt werden.
+ 
+enderio.killerJoe.fuelTank=Nährbarer Tank
+enderio.killerJoe.giveXp.tooltip=Nehme XP
+enderio.killerJoe.giveXp10.tooltip=Take 10 XP-Level
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Nimm 10 XP-Level

File: resources/assets/enderio/lang/de_DE.lang [hunk 15]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
+ 
+tile.blockWeatherObelisk.name=Wetterobelisk
+tile.blockWeatherObelisk.tooltip.detailed.line1=Ändere das derzeitige Wetter
+tile.blockWeatherObelisk.tooltip.detailed.line2=Benötigt eine Menge an Energie
+tile.blockWeatherObelisk.tooltip.detailed.line3=und ein Katalysator-Item
+ 
+tile.blockBuffer.item.name=Item-Puffer
+tile.blockBuffer.power.name=Energiepuffer
+tile.blockBuffer.omni.name=Omnipuffer
+tile.blockBuffer.creative.name=Kreativpuffer
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Kreativer Puffer

File: resources/assets/enderio/lang/de_DE.lang [hunk 16]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
+ 
+tile.blockInventoryPanel.name=Inventar-Panel
+tile.blockInventoryPanel.tooltip.basic.line1=Fernzugriff und Crafting
+tile.blockInventoryPanel.tooltip.detailed.line1=Erlaubt Fernzugriff auf Inventare durch
+tile.blockInventoryPanel.tooltip.detailed.line2=Fernsteureungs-Upgrades in Item-Leitungen.
+tile.blockInventoryPanel.tooltip.detailed.line3=Benötigt Nährbares Destillat als Treibstoff.
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Nahrhaftes Destillat

**Comment by HenryLoenwind** [comment]
BTW: Minecraft's translation uses "Gegenstand" for "item", e.g.

````
inventory.binSlot=Gegenstand zerstören
key.drop=Gegenstand fallenlassen
```

**Comment by wlhlm** [comment]
@HenryLoenwind Yeah, but many other mod's translations use "Item" untranslated. I found "Gegenstand" a little awkward to use. Also, the current EIO translation already uses Item, so I'd say it's established vocabulary ;)

**Comment by wlhlm** [comment]
I have updated the branch according the @XFactHD's feedback. Thanks for having a look at this!

**Comment by HenryLoenwind** [comment]
Ok, makes sense. I'm not usually using the translations when playing, so I had to look it up yesterday when translating Ender IO Addons. Which reminds me, I should probably finish it. Gar, those description texts where a pain to write in the first place, and now I have to do them again...

File: resources/assets/enderio/lang/de_DE.lang [hunk 17]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
+enderio.gui.inventorypanel.tooltip.storage.line3=Klicke um zur Rückgabe zu wechseln
+enderio.gui.inventorypanel.tooltip.filterslot.line1=Filterslot
+enderio.gui.inventorypanel.tooltip.filterslot.line2=Platziere hier konfigurierte Item-Filter
+enderio.gui.inventorypanel.tooltip.filterslot.line3=um zu bestimmen, welche Items angezeigt werden.
+enderio.gui.inventorypanel.tooltip.sort.name_down.line1=Sortiere Items bei Namen
+enderio.gui.inventorypanel.tooltip.sort.name_up.line1=Sortiere Items bei Namen,
+enderio.gui.inventorypanel.tooltip.sort.name_up.line2=beginnend am Ende des Alphabets.
+enderio.gui.inventorypanel.tooltip.sort.mod_down.line1=Sortiere Items nach Mod ID
+enderio.gui.inventorypanel.tooltip.sort.mod_down.line2=Items aus ein und derselben Mod werden nach Namen sortiert
+enderio.gui.inventorypanel.tooltip.sort.mod_up.line1=Sort Items nach Mod ID
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Sortiere

File: resources/assets/enderio/lang/de_DE.lang [hunk 18]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
+enderio.gui.inventorypanel.tooltip.sort.mod_up.line2=Items aus ein und derselben Mod werden nach Namen sortiert,
+enderio.gui.inventorypanel.tooltip.sort.mod_up.line3=beginnend am ende des Alphabets.
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Ende

File: resources/assets/enderio/lang/de_DE.lang [hunk 19]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
 
 //Blocks
 tile.blockDarkSteelPressurePlate.name=Dunkeldruckplatte
 tile.blockDarkSteelPressurePlate.tooltip.detailed.line1=Wird nur von Spielern aktiviert
+tile.blockDarkSteelPressurePlate.tooltip.detailed.line2=Kann in der Färbemaschine gefärbt werden,
+tile.blockDarkSteelPressurePlate.tooltip.detailed.line3=unsichtbar wenn mit Ziemlich Klaren Glas
+tile.blockDarkSteelPressurePlate.tooltip.detailed.line4=gefärbt.
+tile.blockDarkSteelPressurePlate.silent.tooltip.common.line1=Lautlos
+tile.blockDarkSteelPressurePlate.silent.tooltip.detailed.line1=Wird nur durch Spieler aktiviert
+tile.blockDarkSteelPressurePlate.silent.tooltip.detailed.line2=Kann in der Färbemaschine,
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Kann in der Färbemaschine gefärbt werden,

File: resources/assets/enderio/lang/de_DE.lang [hunk 20]
```diff
@@ -278,11 +473,113 @@ item.itemBrokenSpawner.name=Zerstörter Spawner
...
+tile.blockDarkSteelPressurePlate.silent.tooltip.detailed.line3=unsichtbar wenn mit Ziemlich Klaren glas
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Ziemlich Klarem Glas

File: resources/assets/enderio/lang/de_DE.lang [hunk 21]
```diff
@@ -290,11 +587,18 @@ tile.blockTravelAnchor.tooltip.detailed.line1=Erlaubt Teleport zu anderen Ankern
...
 tile.blockTravelAnchor.tooltip.detailed.line4=Kann auch mit dem Stab des Reisens
-tile.blockTravelAnchor.tooltip.detailed.line5=verwendet werden
+tile.blockTravelAnchor.tooltip.detailed.line5=verwendet werden.
+tile.blockTravelAnchor.tooltip.detailed.line6=Kann in einer Färbemaschine gefärbt werden.
 
 enderio.blockTravelPlatform.invalidTarget=Ziel ist blockiert
 enderio.blockTravelPlatform.outOfRange=Ziel ist außer Reichweite
 
+tile.blockTelePad.name=Telepad-Block
+tile.blockTelePad.tooltip.detailed.line1=Benutzt Energie um Entitäten zu einer
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: um Lebewesen zu einer

File: resources/assets/enderio/lang/de_DE.lang [hunk 22]
```diff
@@ -313,19 +629,25 @@ tile.blockPaintedStair.name=Bemalte Treppe
...
 tile.blockPaintedGlowstone.name=Bemaltes Glowstone
+tile.blockPaintedCarpet.name=Bemalter Teppich
+enderio.painter.tooltip.wasPainted=§oWas denkst du, wird das wohl tun?
 
 tile.blockPainter.tooltip.detailed.line1=Verwendet, um folgendes zu bemalen:
-tile.blockPainter.tooltip.detailed.line2= Leitungsfassaden, Treppen, Zäune
-tile.blockPainter.tooltip.detailed.line3= Zauntore, Stufen, Pflastersteinwände
-tile.blockPainter.tooltip.detailed.line3= Glasrahmen
+tile.blockPainter.tooltip.detailed.line2=Leitungsfassaden, Treppen, Zäune,
+tile.blockPainter.tooltip.detailed.line3=Tore, Stufen, Pflastersetinwände,
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Pflastersteinwände

File: resources/assets/enderio/lang/de_DE.lang [hunk 23]
```diff
@@ -336,8 +658,33 @@ item.itemMagnet.tooltip.detailed.line4=in der Hotbar sein, um zu funktionieren
...
+enderio.item.itemSoulVessel.tooltip.fluidname=Flüssigkeitsname:
+enderio.item.itemSoulVessel.tooltip.color=Farbe:
+enderio.item.itemSoulVessel.tooltip.empty=Leer
+item.itemSoulVessel.tooltip.detailed.line1=Rechts-Klick auf einen Mob
+item.itemSoulVessel.tooltip.detailed.line2=um dessen Seele zu fangen
+item.itemSoulVessel.tooltip.detailed.line3=Rechts-Klick um sie wieder freizulasenn
+
+item.itemXpTransfer.name=Erfahrungsstab
+item.itemXpTransfer.tooltip.detailed.line1=Rechts-Klicke auf einen Tank mit
+item.itemXpTransfer.tooltip.detailed.line2=Flüssig-XP, um level zu erlangen
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Level

File: resources/assets/enderio/lang/de_DE.lang (hunk 22 above)
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "Bruchstein" is used in vanilla

File: resources/assets/enderio/lang/de_DE.lang [hunk 24]
```diff
@@ -397,11 +755,39 @@ enderio.darksteel.upgrade.sound.tooltip.detailed.line2=nahen Geräuschquellen
 enderio.darksteel.upgrade.sound.enabled=Schalldetektor Aktiv
 enderio.darksteel.upgrade.sound.disabled=Schalldetektor Inaktiv
 
+enderio.darksteel.upgrade.gogglesOfRevealing.name=Brille der Enthüllung
+enderio.darksteel.upgrade.gogglesOfRevealing.tooltip.detailed.line1=Vis Discount: 5%
+enderio.darksteel.upgrade.gogglesOfRevealing.enabled=Magie wird enthüllt
+enderio.darksteel.upgrade.gogglesOfRevealing.disabled=Magie ist versteckt
+
+enderio.darksteel.upgrade.naturalistEye.name=Naturalists Auge
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Auge des Arboristen

File: resources/assets/enderio/lang/de_DE.lang (hunk 22 above)
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: That's right. --> Bruchsteinwände

File: resources/assets/enderio/lang/de_DE.lang [hunk 25]
```diff
@@ -559,58 +1022,106 @@ enderio.gui.conduit.item.roundRobinEnabled=Round-Robin Aktiviert
...
+enderio.config.anchor=Ankereinstellungen
+enderio.config.anchor.tooltip=Anchors away!
 
-enderio.config.spawner=PoweredSpawner Settings
-enderio.config.spawner.tooltip=I ran out of witty tooltips.
+enderio.config.staff=Stabeinstellungen
+enderio.config.staff.tooltip=Sei vorsichtig mit dem Ding!
+
+enderio.config.darksteel=Dunkelstahl
+enderio.config.darksteel.tooltip=Es nicht nur gewöhnlicher Stahl, es ist §oDunkel§r§estahl.
```
Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Es ist ...

File: resources/assets/enderio/lang/de_DE.lang (hunk 21 above)
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: I think it also teleports dropped items, XP orbs, minecarts and such things. As such, I'd leave it at "Entitäten", unless there's better better word in use somewhere?

Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Sorry, I didn't know that, I never used the TelePad. I think we should find another translation as Entitäten is a bit too scientific in my opinion.

Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: I haven't used it either, it isn't even finished yet. @tterrag1098 will it stay at teleporting entities or be restricted to mobs and players?

File: resources/assets/enderio/lang/de_DE.lang [hunk 26]
```diff
@@ -347,12 +694,18 @@ item.darkSteel_boots.name=Dunkelstiefel
 item.darkSteel_sword.name=Der Ender
 item.darkSteel_pickaxe.name=Dunkelspitzhacke
 item.darkSteel_axe.name=Dunkelaxt
+item.darkSteel_shears.name=Dunkelschere
+
+enderio.item.darkSteel.tooltip.lvs=level
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: Level

File: resources/assets/enderio/lang/de_DE.lang [hunk 27]
```diff
@@ -347,12 +694,18 @@ item.darkSteel_boots.name=Dunkelstiefel
...
 
 enderio.item.darkSteel_pickaxe.tooltip.effPowered=Effizienz, wenn aktiviert
 enderio.item.darkSteel_pickaxe.tooltip.effObs=Effizienz baut Obsidian ab
+enderio.item.darkSteel_pickaxe.tooltip.cost=Kosten
 
 enderio.item.darkSteel_axe.tooltip.multiHarvest=Schleiche, um Mehr Holz auf einmal abzubauen
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: while we're at it, even if it's not part of your changes: "mehr"

File: resources/assets/enderio/lang/de_DE.lang [hunk 28]
```diff
@@ -383,8 +736,13 @@ enderio.darksteel.upgrade.empowered_four.tooltip.detailed.line1=Energie: $P
...
 
-enderio.darksteel.upgrade.jump_one.name=Springen
-enderio.darksteel.upgrade.jump_one.tooltip.detailed.line1=Erhöht Schreit- und Sprunghöhe
+enderio.darksteel.upgrade.travel.name=Reise
+enderio.darksteel.upgrade.travel.tooltip.detailed.line1=Halte Shift und Rechts-Klick zum Teleportieren,
+enderio.darksteel.upgrade.travel.tooltip.detailed.line2=oder aktiviere einen Reiseanker oder Ender-IO
+
+enderio.darksteel.upgrade.spoon.name=Löffel
+enderio.darksteel.upgrade.spoon.tooltip.detailed.line1=Wer braucht eine Schaufel 
+enderio.darksteel.upgrade.spoon.tooltip.detailed.line2=wenn man einen Löffel hat?
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "man"->"er" ("man" doesn't work with the "Wer" at the beginning)

File: resources/assets/enderio/lang/de_DE.lang [hunk 29]
```diff
@@ -397,11 +755,39 @@ enderio.darksteel.upgrade.sound.tooltip.detailed.line2=nahen Geräuschquellen
...
+
+enderio.darksteel.upgrade.apiaristArmor.boots.name=Imkerschuhe
+enderio.darksteel.upgrade.apiaristArmor.boots.tooltip.detailed.line1=Schutz gegen Beinen
+
 item.itemGliderWing.name=Gleiterflügel
 item.itemGliderWings.name=Gleiterflügelpaar
 item.itemGliderWings.tooltip.detailed.line1=Upgrade für den
 item.itemGliderWings.tooltip.detailed.line2=Dunkelbrustpanzer
 
+enderio.darksteel.upgrade.jump.enabled=Springen Aktiviert
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "aktiviert" (same for all the (de-)"aktiviert"s that follow)

File: resources/assets/enderio/lang/de_DE.lang [hunk 30]
```diff
@@ -422,15 +810,27 @@ enderio.darksteel.upgrade.speed_three.tooltip.detailed.line1=Erhöht Bewegungsge
 enderio.darksteel.upgrade.swim.name=Schwimmflossen
 
 enderio.darksteel.upgrade.nightVision.name=Nachtsicht
-enderio.darksteel.upgrade.nightVision.enabled=Nachtsicht Aktiv
-enderio.darksteel.upgrade.nightVision.disabled=Nachtsicht Inaktiv
+enderio.darksteel.upgrade.nightVision.enabled=Nachtsicht Aktiviert
+enderio.darksteel.upgrade.nightVision.disabled=Nachtsicht Deaktiviert
+
+enderio.darksteel.upgrade.solar_one.name=Solar I
+enderio.darksteel.upgrade.solar_one.tooltip.detailed.line1=Generiert Energie von der Sonne.
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "von der Sonne"->"aus Sonnenlicht"

File: resources/assets/enderio/lang/de_DE.lang [hunk 31]
```diff
@@ -440,12 +840,46 @@ enderio.gui.tooltip.redstoneControlMode.on=Aktiv mit Signal.
...
+enderio.gui.machine.button.storeall=Speicher gesamte
+enderio.gui.machine.tooltip.storeall=Spieler-XP
+enderio.gui.machine.button.retrievelevel=Entnehme 1
+enderio.gui.machine.tooltip.retrievelevel=Level Spieler-XP
+enderio.gui.machine.button.retrievelevels=Entnehme 10
+enderio.gui.machine.tooltip.retrievelevels=Level Spieler-XP
+enderio.gui.machine.button.retrieveall=Entnehme gesamte
+enderio.gui.machine.tooltip.retrieveall=Spieler-XP
 enderio.gui.machine.ioMode.overlay.tooltip=IO Konfigurieren
+enderio.gui.machine.ioMode.overlay.tooltip.visible.line1=Shift-Klick um alle Seiten auf den
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "auf"->"in"

File: resources/assets/enderio/lang/de_DE.lang [hunk 32]
```diff
@@ -530,6 +976,18 @@ enderio.gui.travelAccessable.protected=Geschützt
 enderio.gui.travelAccessable.unauthorised=Dir ist es nicht erlaubt, auf diesen Block zuzugreifen.
 enderio.gui.travelAccessable.enterCode=Zugriffscode benötigt
 enderio.gui.travelAccessable.ok=Autorisieren
+enderio.gui.travelAccessable.skipLocked=Überspringe abgeschlossenen Anker, benutzte gezielten Teleport.
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "abgeschlossene Anker" (use plural), same for the two lines below

File: resources/assets/enderio/lang/de_DE.lang [hunk 33]
```diff
@@ -530,6 +976,18 @@ enderio.gui.travelAccessable.protected=Geschützt
...
+enderio.gui.travelAccessable.skipObstructed=Überspringe versperrten Anker.
+enderio.gui.travelAccessable.skipPrivate=Überspringe privaten Anker.
+
+enderio.gui.telepad.configure.travel=Reiseeinstellungen
+enderio.gui.telepad.configure.telepad=Telepad-Einstellungen
+enderio.gui.telepad.blocked=Teleport blockiert!
+
+//Spawn Gurad [sic]
+enderio.gui.spawnGurad.showRange=Zeige Reichweite
+enderio.gui.spawnGurad.hideRange=Verstecke Reichweite
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "Verberge" (Verstecken is only "hide" in the sense of putting something in a secret place where you don't want to to be found, where verbergen is taking something out of sight)

File: resources/assets/enderio/lang/de_DE.lang [hunk 34]
```diff
@@ -559,58 +1022,106 @@ enderio.gui.conduit.item.roundRobinEnabled=Round-Robin Aktiviert
...
+enderio.gui.weather.task.clear=Klares Wetter
+enderio.gui.weather.task.rain=Regnerisches Wetter
+enderio.gui.weather.task.storm=Gewitter
+enderio.gui.weather.requireditem=Benötigtes Item: %s
 
-enderio.config.power=Power Settings
-enderio.config.power.tooltip=So many numbers to change...
+//Enchants
+enchantment.enderio.soulBound=Seelengebunden
+description.enchantment.enderio.soulBound=Verhindert das Verlieren des Item bei Tod
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: "beim Sterben" (Tod is the state of being dead, Sterben is the act of getting there)

File: resources/assets/enderio/lang/de_DE.lang [hunk 35]
```diff
@@ -559,58 +1022,106 @@ enderio.gui.conduit.item.roundRobinEnabled=Round-Robin Aktiviert
...
 
-enderio.config.magnet=Magnet Settings
-enderio.config.magnet.tooltip=How do they work?
+enderio.config.personal=Persönliche Einstellungen
+enderio.config.personal.tooltip=Ich verspreche, ich werde niemandem davon erzählen.
 
-enderio.config.fluid=Fluid Settings
-enderio.config.fluid.tooltip=Pure liquid energy!
+enderio.config.anchor=Ankereinstellungen
+enderio.config.anchor.tooltip=Anchors away!
```
Comment (HenryLoenwind) [inline resources/assets/enderio/lang/de_DE.lang]: Anker los! Setzt die Segel!

File: resources/assets/enderio/lang/de_DE.lang (hunk 21 above)
Comment (wlhlm) [inline resources/assets/enderio/lang/de_DE.lang]: Yeah, I wasn't too sure about this one.

The Minecraft Wiki [translates "Entity" with "Objekt"](http://minecraft-de.gamepedia.com/Objekt), maybe that's a better one.

**Comment by wlhlm** [comment]
Updated again.

Thanks for the feedback!

File: resources/assets/enderio/lang/de_DE.lang (hunk 21 above)
Comment (Vexatos) [inline resources/assets/enderio/lang/de_DE.lang]: Oh yea, this amazing issue again. Uuuh... I translated the word multiple times but I can't remember what I translated it to. I think I based it on the context, sometimes it was "Lebewesen" for instance, but not all the time. I also recall changing the English translation instead to not contain this word. It's horrible.

Comment (Vexatos) [inline resources/assets/enderio/lang/de_DE.lang]: Okay, here is a cheat I found in my Carpenter's Blocks translation: "Benutzt Energie, alles mögliche zu anderen Orten und Dimensionen zu teleportieren."

What do you think?

Comment (XFactHD) [inline resources/assets/enderio/lang/de_DE.lang]: Would be a viable solution. "Objekt" is probably to generic.

Comment (wlhlm) [inline resources/assets/enderio/lang/de_DE.lang]: Ok, I'll go with @Vexatos's suggestion.

**Comment by wlhlm** [comment]
Update!

**Pull request merged**